### ISSUE TITLE
Make help menu items specific to each map

### ISSUE THREAD

**Issue by reginapramesti** [TerriaJS/terriajs#4161]
Currently the help menu items are defined in terriajs rather than on individual maps. This will restrict us from using different contents for different maps. Ideally we want this to be different for each map as different maps bring different challenges and use cases (e.g. NSW DT users would require help with 3D navigation, whereas DEA users would require help with satellite imagery).

**Comment by zoran995** [comment]
I have one question from internationalization part, will it be possible to provide help items in multiple languages depending on the current language of map. Internationalization part provides the possibility to have a multilanguage map, so it would be the desired approach to have help content in the current map language. For example, the map provided in Serbian and English language should have help content in both languages.

**Comment by reginapramesti** [comment]
Hi @zoran995,

Currently we have made it so that the help items are loaded in from `config.json`, so it will be possible for developers to change the content without having to rebuild the map. Though not in multiple languages yet unfortunately, as it's a new feature we've only recently implemented. 

The internationalisation is definitely something we would like to work towards so I've created an issue for it https://github.com/TerriaJS/terriajs/issues/4278. We'll let you know on the new issue once we start working on it. Alternatively, feel free to have a crack at implementing the internationalisation if you'd like. :) 

Cheers,
Regina
